### PR TITLE
Improve error message for mesheryctl mesh

### LINKED ISSUE #6270: [mesheryctl] improve the error messages shown to users for "mesheryctl mesh"
**Description**: Improve the error messages in mesheryctl commands from the user's perspective. 
**Commands in scope of this issue**: `mesheryctl mesh`, `mesheryctl mesh validate`, `mesheryctl mesh deploy`

---

We need to make sure of the following things
- There are proper error messages when users
  - Give more arguments to a command than required
  - Misspell a command
  - Do not give the required input via flags or args
  - Use the command without any arguments or flags.
- The error messages being displayed to users are meaningful from user's perspective. And if applicable, a potential fix for the problem should be shown too.  For example
  - "failed to unmarshal response body" could be modified to "couldn't process JSON response from server"
  - "manifest not found" could be modified to "manifest not found. Please provide the path to the app manifest."
  
 --- 
  
   📚 For the **Reference** head over to this [pr](https://github.com/meshery/meshery/pull/5661).

### PR DESCRIPTION
**Pull request by PratikforCoding**
Signed-off-by: PratikforCoding <kotalpratik@gmail.com>

**Notes for Reviewers**

This PR fixes (partial) #6270 

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by PratikforCoding** [comment]
@nishantwrp review this please.

**Comment by nishantwrp** [comment]
This was covered by #6626 along with some other commands.

File: mesheryctl/internal/cli/root/mesh/mesh.go [hunk 1]
```diff
@@ -28,7 +28,7 @@ mesheryctl mesh
 		}
 
 		if ok := utils.IsValidSubcommand(availableSubcommands, args[0]); !ok {
-			return errors.New(utils.MeshError(fmt.Sprintf("'%s' command not found.  Use 'mesheryctl mesh --help' to display usage guide.'\n ", args[0])))
+			return errors.New(utils.MeshError(fmt.Sprintf("'%s' is a invalid command. Use 'mesheryctl mesh --help' to display usage guide.\n ", args[0])))
```
Comment (leecalcote) [inline mesheryctl/internal/cli/root/mesh/mesh.go:31]:
```suggestion
			return errors.New(utils.MeshError(fmt.Sprintf("'%s' is an invalid command. Use 'mesheryctl mesh --help' to display usage guide.\n ", args[0])))
```

**Comment by leecalcote** [comment]
Adding " (partial) " to PR description.

**Comment by leecalcote** [comment]
Thanks, @PratikforCoding !

**Review by leecalcote — APPROVED** [review]

**Comment by PratikforCoding** [comment]
Thanks @leecalcote sir!

**Pull request merged**